qa/entrypoint.sh

```diff
@@ -22,6 +22,7 @@ if [[ $TRITON_ENABLE_GPU != "OFF" ]]
 then
   echo 'Running tests for GPU models...'
   MODEL_REPO="${QA_DIR}/L0_e2e/model_repository" "$TEST_SCRIPT"
+  sleep 3
```

Comment:
Could you add a comment detailing exactly why this is necessary?

Reply:
This change is no longer in effect.